<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_register">
        <property name="text">
         <string>Cadastrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_return">
        <property name="text">
         <string>Voltar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="title_cadastro">
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Cadastro</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="lb_card_number">
        <property name="text">
         <string>Número do Cartão
de Crédito</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_card_number"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_security_number_card">
        <property name="text">
         <string>Código
de Segurança do
Cartão de Crédito</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_cpf"/>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rd_customer">
          <property name="text">
           <string>Comprador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rd_game_manager">
          <property name="text">
           <string>Gestor de Jogos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_security_number_card">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_cpf">
        <property name="text">
         <string>CPF</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_expiration_date_card">
        <property name="text">
         <string>Data de
Validade do Cartão
de Crédito
Formato: 08/25</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_password">
        <property name="text">
         <string>Senha</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_authority">
        <property name="text">
         <string>Privilégio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="cb_month">
          <item>
           <property name="text">
            <string>01</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>03</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>04</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>05</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>06</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>07</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>09</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_year">
          <item>
           <property name="text">
            <string>2019</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2020</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2021</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2022</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2023</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2024</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2025</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2026</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_cpf</tabstop>
  <tabstop>line_password</tabstop>
  <tabstop>line_card_number</tabstop>
  <tabstop>line_security_number_card</tabstop>
  <tabstop>rd_customer</tabstop>
  <tabstop>rd_game_manager</tabstop>
  <tabstop>btn_register</tabstop>
  <tabstop>btn_return</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
